### PR TITLE
[MXNET-617] - Improve Clojure styles with cljfmt tools revise

### PR DESCRIPTION
**Pull request by agilecreativity**
## Description ##

Use [cljfmt](https://github.com/weavejester/cljfmt) to keep styles of the project consistent.
This include the updated version that have a cleaner commit history and rebase on the latest changes from @gigasquid.

Please help review.
Thanks

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ X] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ X] Changes are complete (i.e. I finished coding on this PR)
- [ X] All changes have test coverage:
- [X ] Code is well-documented: 
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ X] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [X] Remove extra file `vi ci-test.sh` from the project
- [X] Add steps to run the new script to README.md file
- [X] Add lein-cljfmt to the plugins vector for all Clojure projects
- [X] Add the `lein-cljfmt-check` to each project and the base directory to be run in batch
- [X] Run the `lein-cljfmt-fix` to fix the style of the project excluding some files when necessary

## Comments ##

- These scripts will make it easier for new contributor to keep using the same styles going forward
- The bash flag from `lein-cljfmt-check` is intentionally omit the `-e` to allow the script to continue

### CODE REVIEW

File: contrib/clojure-package/examples/cnn-text-classification/lein-cljfmt-check [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env sh
```
Comment (marcoabreu) [inline contrib/clojure-package/examples/cnn-text-classification/lein-cljfmt-check]: What do you think about moving this script to a bit more top-level directory and then applying it recursively?

Comment (agilecreativity) [reply contrib/clojure-package/examples/cnn-text-classification/lein-cljfmt-check]: @marcoabreu 
I did not think of this, but yes this is a great idea indeed.

**Comment by agilecreativity** [comment]
@gigasquid 
Please help review and provide any feedback for improvement.
Thanks

**Comment by gigasquid** [comment]
Thanks @agilecreativity for putting this together. I'll take a look later tonight.

**Pull request merged**